manifests/stage-nv-peer-mem-driver/0010_nv-peer-mem-driver-ds.yaml

```diff
@@ -108,9 +111,12 @@ spec:
             initialDelaySeconds: 10
             failureThreshold: 1
       volumes:
-        - name: run-mlnx-ofed
+        - name: ofed-usr-src
+          hostPath:
+            path: {{ .CrSpec.OFEDDriverSourcePath }}/usr/src
+        - name: ofed-lib-modules
           hostPath:
-            path: /run/mellanox/drivers
+            path: {{ .CrSpec.OFEDDriverSourcePath }}/lib/modules
```

Comment:
so like we said you can know what should be the value per of OFED deploy flag

Reply:
done